### ISSUE TITLE
Data corrupted in pool with raid2

### ISSUE THREAD

**Issue by linuxbest-tp** [zfsonlinux/zfs#9623]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| Ubuntu 16.04 LTS
Distribution Version	| Ubuntu 16.04 LTS
Linux Kernel	| 4.4.0-21-generic #37-Ubuntu SMP
Architecture	| 
ZFS Version	| 0.6.5.6-0ubuntu3
SPL Version	| 0.6.5.6-0ubuntu1

### Describe the problem you're observing

copy data from old freebsd pool to new server, but found data corrupted in new pool. this is important office data, please advice.

root@srvbig:~# zpool status
  pool: ZPOOL
 state: ONLINE
status: One or more devices has experienced an error resulting in data
	corruption.  Applications may be affected.
action: Restore the file in question if possible.  Otherwise restore the
	entire pool from backup.
   see: http://zfsonlinux.org/msg/ZFS-8000-8A
  scan: resilvered 1.89M in 0h0m with 0 errors on Mon Nov 24 17:18:05 2019
config:

	NAME        STATE     READ WRITE CKSUM
	ZPOOL       ONLINE      14 43.5K   251
	  raidz2-0  ONLINE      36 87.0K   502
	    sda     ONLINE      48 24.5K     0
	    sdb     ONLINE      39 21.7K     0
	    sdc     ONLINE      12 21.7K     0
	    sdd     ONLINE       4 24.2K     0
	    sde     ONLINE      39 21.6K     0
	    sdf     ONLINE       4 21.6K     0
	    sdg     ONLINE       4 24.5K     0
	    sdh     ONLINE       4 21.8K     0
	    sdi     ONLINE      35 21.8K     0
	    sdl     ONLINE       0 24.6K     0
	    sdm     ONLINE       0 21.9K     0
	    sdn     ONLINE      38 27.7K     0

errors: 1226 data errors, use '-v' for a list

### Describe how to reproduce the problem
write office data to new ubuntu pool, corruption occurs
### Include any warning/errors/backtraces from the system logs

no log emitted in systems

**Comment by kpande** [comment]
check your new hardware for defects, scrub pools regularly, and use the mailing list for support questions like this.

**Comment by Ornias1993** [comment]
I agree with @kpande This is not a bug or a feature but more of a support request.
It doesn't contain enough information to call it a bug.

Please close the

**Comment by devZer0** [comment]
>0.6.5.6

why are you using that old zfs version on a new server?

that's  more then 3 years old and outdated

please use 0.8.2 or at least 0.7.13
